### DISCUSSION TITLE
Why? This is not a social media app. It does not need a "feed".

### DISCUSSION THREAD

**Discussion by Mugane** [community/community#125165] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Product Feedback

### Body

Yet another unwanted distraction. Please provide a means to get rid of it. Please.

**Comment by ankit-orion** [comment]
I get where you're coming from. The "feed" feature on GitHub is meant to help users stay updated with the latest activities like commits, pull requests, and issues from repositories they follow. It's supposed to make collaboration easier and keep everyone in the loop without having to check each repo individually.

But I hear you—GitHub is primarily for coding, and not everyone wants a "feed" like on social media. I'll share your feedback with the team. We're always trying to find the right balance to keep things useful and not cluttered.

Thanks for sharing your thoughts!
